core/templates/dev/head/pages/exploration_player/AudioTranslationManagerService.js

```diff
@@ -138,6 +175,13 @@ oppia.factory('AudioTranslationManagerService', [
       },
       clearCurrentAudioLanguageCode: function() {
         _currentAudioLanguageCode = null;
+      },
+      isAutogeneratedLanguageCodeSelected: function() {
+        return _autogeneratedLanguageCodeIsSelected;
+      },
+      getSpeechSynthesisLanguageCode: function() {
+        return LanguageUtilService.getAutogeneratedAudioLanguage(
+          _explorationLanguageCode).speech_synthesis_code;
```

Comment:
Should make domain object and use speechSynthesisCode

Reply:
done